developer-docs/graph/README.md

```diff
@@ -1,285 +1,155 @@
+Ensure you have the following installed:
+- [Node.js](https://nodejs.org)
+- [Docker](https://docs.docker.com/get-docker/)
 
-The application receives its configuration from the environment. Each method of launching the app has its own source for the environment. If you run a container image using Kubernetes, it is likely your environment injection will be configured in a Helm chart. For local Docker-based development, you may specifiy the environment or point to an environment file (the included [docker-compose.yaml](./docker-compose.yaml) can has a self-contained environment which may be edited to suit your purposes). If running natively using the script included in `package.json`, the app will use a local `.env` file.
+### Quick Start
 
-Environment files are documented [here](./ENVIRONMENT.md), and a sample environment file is provided [here](./env.template).
+1. Clone the repository and navigate to the project directory:
+   ```bash
+   git clone https://github.com/ProjectLibertyLabs/gateway.git
+   cd gateway
+   ```
 
-1. Copy the template values into the .env file.
-   ```sh
-   cp env.template .env
+2. Install dependencies:
+   ```bash
+   npm install
    ```
-2. Replace template values with values appropriate to your environment.
 
-### Usage
+3. Set up environment variables:
+   ```bash
+   cp env-files/graph.template.env .env
+   ```
+   Edit the `.env` file with appropriate values for your environment.
 
-Note: using [docker compose file](docker-compose.yaml) to start the services. This will start the services in development mode.
+4. Start all services using Docker Compose:
+   ```bash
+   docker compose up -d redis frequency graph-service-api graph-service-worker
+   ```
 
-#### 1. Start the Redis server container and the Frequency container. You can view the logs with your Docker setup.
+### Local Development (without containers)
 
-```sh
-docker compose up -d redis frequency
-```
+If you prefer to run services locally for development:
 
-#### 2. Once [Frequency](https://polkadot.js.org/apps/?rpc=ws%3A%2F%2F127.0.0.1%3A9944#/explorer) is up. Run a graph setup with Alice as provider 1 and 2,3,4,5,6 as users.
+1. Start Redis and Frequency:
+   ```bash
+   docker compose up -d redis frequency
+   ```
 
-```sh
-make setup
-```
+2. Set up the graph with test data:
+   ```bash
+   make setup-account
+   ```
 
-#### 3. Run the following command to start the graph service api and worker containers. This will start the account service api and worker in development mode.
+3. Start the Graph Service API:
+   ```bash
+   make start-graph-api
+   ```
 
-```sh
-docker compose up -d api worker
-```
+4. Start the Graph Service Worker:
+   ```bash
+   make start-graph-worker
+   ```
 
-#### 4. Check the job in [BullUI](http://0.0.0.0:3000/queues/), to monitor job progress based on defined tests.
+### Verification
 
-## 📋 Testing
+- Access Swagger UI: [http://localhost:3000/docs/swagger](http://localhost:3000/docs/swagger)
```

Comment:
the ports would be different when we run it inside docker

Reply:
Good catch. I updated the documentation.